openrpc/src/examplePairingObjects/GetTransactionsPairs.json

```diff
@@ -0,0 +1,22 @@
+{
+  "SuccessfulGetTransactionsRequest": {
+    "name": "Successful getTransactions Request",
+    "description": "Query a list of transactions starting from the given startLedger.",
+    "params": [
+      { "$ref": "#/components/examples/GetTransactionsRequest" }
```

Comment:
This is where you'll probably want to use a couple of `$ref` objects for starting ledger, pagination with cursor, etc.